migrations/versions/9ede8e012999_enable_fastseg_models.py

```diff
@@ -0,0 +1,40 @@
+"""Enable fastseg models to appear in the OMZ table
+Revision ID: 9ede8e012999
+Revises: f7fcb58b99cd
+Create Date: 2022-02-11 15:23:34.127963
+"""
+
+"""
+ OpenVINO DL Workbench
+ Migration: Enable fastseg models to appear in the OMZ table
+ Copyright (c) 2022 Intel Corporation
+ LEGAL NOTICE: Your use of this software and any required dependent software (the “Software Package”) is subject to
+ the terms and conditions of the software license agreements for Software Package, which may also include
+ notices, disclaimers, or license terms for third party or open source software
+ included in or with the Software Package, and your use indicates your acceptance of all such terms.
+ Please refer to the “third-party-programs.txt” or other similarly-named text file included with the Software Package
+ for additional details.
+ You may obtain a copy of the License at
+      https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9ede8e012999'
+down_revision = 'f7fcb58b99cd'
```

Comment:
Have you checked the order?

Reply:
Yes, the order has already been adjusted to that of master